RELEASES_CHECKLIST.md

```diff
@@ -55,16 +49,12 @@ fork for now.
 1. Open a release PR
     - Wait for approvals from Core team members
     - Ensure the entire CI pipeline is green
-1. Execute `cargo unleash de-dev-deps` in the ink! repository.
-    - We cannot release with `dev-dependencies` due to some cyclic dependencies that
-      exist.
-1. Do a dry run with `cargo release [level] -v --no-tag --no-push --allow-dirty`
+1. Do a dry run with `cargo release [level] -v --no-tag --no-push`
```

Comment:
What values can `[level]` have?

Reply:
You can find them as part of the [`cargo-release`](https://github.com/crate-ci/cargo-release/blob/master/docs/reference.md) documentation. I don't think it makes sense to have them here, but if you think it would be helpful I don't adding them

Reply:
Lol https://github.com/paritytech/ink/pull/1412